net/netavark/Makefile

```
@@ -0,0 +1,51 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=netavark
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/containers/netavark.git
+PKG_SOURCE_DATA:=2023-05-12
```

```suggestion
PKG_SOURCE_DATE:=2023-05-12
```

data -> date

small typo error ;)

---

Thanks. Fixed as is.